modules/ROOT/pages/index.adoc

```diff
@@ -6,34 +6,23 @@ endif::[]
 
-
-== Business Solution
-
-Growing business demands for diverse deployment environments introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
-
-In a common enterprise setup, numerous teams are engaged in building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is a siloed environment, with teams managing these applications in different ways. 
-
-With Anypoint Service Mesh, you can connect your non-Mule applications, data, and devices through APIs in an application network. You can manage, govern, and secure your application network using Anypoint Platform.
-
-When your microservices can exist outside of the MuleSoft ecosystem, built using different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
-
-Most importantly, Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. You can discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
-
 == Anypoint Service Mesh Architecture
 
-A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job. This allows for providing security to your services. The manner in which these sidecar proxies interact is governed by an overall mixer, where a user can apply different policies and security.
+A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
+
+The sidecar proxy enables you to bring your existing non-Mule microservices from the Kubernetes Cluster as APIs into Anypoint Platform. You can then manage these APIs within API Designer, API Manager, and Exchange. Additionally, you can monitor the health of your APIs using API analytics.
 
-The mixer is a highly modular and extensible component, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.
+A mixer, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.
```

Comment:
```suggestion
A _mixer_, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform enable the rest of Istio to be agnostic of the backend processing.
```